packages/web-components/src/radio-group/README.md

```diff
@@ -0,0 +1,127 @@
+# Radio Group
+
+> RadioGroup lets people select a single option from two or more Radio items. Use RadioGroup to present all available choices if there's enough space. For more than 5 choices, consider using a different component such as Dropdown.
+
+<br />
+
+## **Design Spec**
+
+[Link to Radio Design Spec in Figma](https://www.figma.com/file/4XWsJrlpEcuEpUnZbtoIBU/Radio?node-id=1295%3A0&t=2wXnjT4ybxIxT6wu-0)
+
+<br />
+
+## **Engineering Spec**
+
+A radio group is a set of checkable buttons, known as radio buttons, where no more than one of the buttons can be checked at a time. Some implementations may initialize the set with all buttons in the unchecked state in order to force the user to check one of the buttons before moving past a certain point in the workflow.
+
+### Use Case
+
+Radio group allows the user to be presented with a list of all the options visible which can facilitate the comparison of choice.
+
+<br />
+
+## Class: `RadioGroup`
+
+<br />
+
+### **Component Name**
+
+<br />
+
+`fluent-radio-group`
+
+<br />
+
+### **Fields**
+
+| Name          | Privacy | Type                     | Default      | Description                                                                                           |
+| ------------- | ------- | ------------------------ | ------------ | ----------------------------------------------------------------------------------------------------- |
+| `disabled`    | public  | `boolean`                | `false`      | Disables the radio group and child radios.                                                            |
+| `name`        | public  | `string`                 |              | The name of the radio group. Setting this value will set the name value for all child radio elements. |
+| `value`       | public  | `string`                 |              | The value of the checked radio.                                                                       |
+| `orientation` | public  | `horizontal \| vertical` | `horizontal` | The orientation of the group                                                                          |
+| `childItems`  | public  | `HTMLElement[]`          |              | The orientation of the group                                                                          |
```

Comment:
What is `childItems`? This looks copied over, but I think this is internals... if you were to pass a property of childItems I don't think it works. IIRC, you slot the items in the default slot.